installer/install-playos/install-playos.py

```diff
@@ -40,9 +41,15 @@ def find_device(device_path):
     boot_device = get_blockdevice("/iso")
     if boot_device is None:
         print("Could not identify installer medium. Considering all disks as installation targets.")
-        available_devices = all_devices
-    else:
-        available_devices = [device for device in all_devices if not device.path.startswith(boot_device)]
+
+    def device_filter(dev):
+        is_boot_device = (boot_device) and (dev.path.startswith(boot_device))
+        is_read_only = dev.readOnly
+        is_big_enough = gb_size(dev) > REQUIRED_INSTALL_DEVICE_SIZE_GB
+
+        return (not is_boot_device) and (not is_read_only) and (is_big_enough)
+
+    available_devices = [device for device in all_devices if device_filter(device)]
 
     print(f"Found {len(available_devices)} possible installation targets:")
```

Comment:
Maybe before this we could print the required disk space, so it's clear why certain devices listed above are not listed below.